shm/inc/ship.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <string>
+#include <memory>
+#include <vector>
```

Comment:
Includes should be sorted alphabetically,so string should be between memory and vector.

Reply:
Sure! Done.